docs/introduction.rst

```diff
@@ -19,42 +31,82 @@ can be open source or proprietary (although some distribution services, like
+  time packaging applications to make the distribution suitable for the end user, instead of focusing
+  on their end goals. This delays the progress of each distribution.
+
+Flatpak strives to fix the issues listed above, by conveniently enabling developers to distribute
+applications from one source and to target the entire Linux desktop.
+
 Reasons to use Flatpak
 ----------------------
 
-Flatpak has some major advantages over other approaches to distributing
-applications on Linux. First and foremost, Flatpak allows a single application
-build to be installed and run on virtually any Linux distribution. It can also
-be used in combination with `Flathub <https://flathub.org/>`_, a centralized
-service for distributing applications on all distributions. This makes it
-possible for application developers to target the entire Linux desktop market
-from one place.
+Flatpak has some major advantages over most system package managers:
+
+- **Universality**: Flatpak allows applications to be installed and run on virtually any Linux
+  distribution. This includes non-GNU distributions, systemd-free distributions,
+  distributions with a read-only operating system (OS), and systems using a different architecture.
+- **Space for innovations**: Flatpak facilitates distribution maintainers to focus on their goals
+  to innovate their distribution.
+- **Stability**: Breakage in a Flatpak application will not risk the system from breaking.
+  This is because Flatpak applications and runtimes are contained to not interfere
+  with the system altogether.
+- **Rootless install**: Elevated privileges are not required when installing a Flatpak
+  application or a runtime.
 
-Flatpak also offers native integration for the main Linux desktops, so that
-users can easily browse, install, run and use Flatpak applications through
-their existing desktop environment and tools.
+Flatpak has some major advantages over other universal approaches to distributing
+applications on Linux:
+
+- **Centralized store**: It can also be used in combination with `Flathub <https://flathub.org/>`_,
+  a centralized service for distributing applications on all distributions. This
+  makes it possible for application developers to target the entire Linux desktop
+  market from one place.
+- **Decentralized by design**: While Flatpak does provide a centralized service for distributing
+  applications, it also allows decentralized hosting and distribution, so that
+  application developers or downstreams can host their own applications and
+  application repositories.
+- **Desktop integration**: Flatpak also offers native integration for the main Linux desktops, so that
+  users can easily browse, install, run and use Flatpak applications through
+  their existing desktop environment and tools.
+- **Space efficiency**: Flatpak links identical runtime and BaseApp versions to save megabytes
+  or even gigabytes worth of storage depending on the amount of applications installed.
```

Comment:
Maybe worth mentioning delta updates? (Only changed files are downloaded for updates)